internal/tests/testpulse/test_config-opts.go

```diff
@@ -5,9 +5,9 @@ import (
 	"github.com/bitcoin-sv/pulse/internal/tests/testrepository"
 )
 
-// WithoutApiAuthorization disables authorization.
+// WithApiAuthorization allows to not use authorization in Pulse.
 func WithoutApiAuthorization() ConfigOpt {
```

Comment:
It is `Without` - means that authorization is disabled.
Maybe bettern name would be `WithApiAuthorizationDisabled`

Reply:
It wasn't added by my, I see that my tool changed comment to different name. But anyway I switched to your recommendation:
**WithApiAuthorizationDisabled**